modules/electrophysiology_browser/jsx/react-series-data-viewer/src/series/components/EEGMontage.tsx

```diff
@@ -303,11 +305,15 @@ const EEGMontage = (
                 mouseDown={dragStart}
                 mouseUp={dragEnd}
                 mouseLeave={dragEnd}
+                chunksURL={chunksURL}
               >
                 <Montage3D />
               </ResponsiveViewer>
             :
-              <ResponsiveViewer>
+              <ResponsiveViewer
+                // @ts-ignore
```

Comment:
Why is this being ignored? If ts flags something it's usually for a reason.

Reply:
I am not sure why. It was not introduced by this PR (or myself) and there seem to be many instances of `// @ts-ignore` throughout the module

Reply:
Huh? This line is being introduced by you in this PR.

Reply:
You're right. However it is also present 11 lines above, as well as in the other `<ResponsieViewer>` component in `SeriesRenderer.tsx`.

It seems to be related to `export default withParentSize(ResponsiveViewer);` at the end of `ResponsiveViewer.tsx`.

Here is the error I get when I remove the `// @ts-ignore` line you pointed out:
<img width="1435" alt="Screenshot 2024-02-27 at 3 17 54 PM" src="https://github.com/aces/Loris/assets/15801528/1cf11cd6-37b5-4eda-b573-f85d355a46c2">

Reply:
That sounds like a real error that needs to be investigated.

Reply:
I believe I found an appropriate solution. It has been pushed to this PR.